### ISSUE TITLE
Linux + ffmpeg 2.8.6, import cv2 fails

### ISSUE THREAD

**Issue by hajapy** [conda-forge/opencv-feedstock#82]
I noticed that the current conda-forge build fails (linux) if ffmpeg 2.8.6 is installed. No such issue on OS X.  

```
In [1]: import cv2
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-72fbbcfe2587> in <module>()
----> 1 import cv2

ImportError: libavcodec.so.57: cannot open shared object file: No such file or directory
```

```
conda list

blas                      1.1                    openblas    conda-forge
bzip2                     1.0.6                         1    conda-forge
ca-certificates           2017.4.17                     0    conda-forge
cairo                     1.14.6                        4    conda-forge
certifi                   2017.4.17                py36_0    conda-forge
dbus                      1.10.10                       3    conda-forge
decorator                 4.0.11                   py36_0    conda-forge
expat                     2.1.0                         2    conda-forge
ffmpeg                    2.8.6                         5    conda-forge
fontconfig                2.12.1                        4    conda-forge
freetype                  2.7                           1    conda-forge
gettext                   0.19.7                        1    conda-forge
giflib                    5.1.4                         0    conda-forge
glib                      2.51.4                        0    conda-forge
gst-plugins-base          1.8.0                         0    conda-forge
gstreamer                 1.8.0                         2    conda-forge
harfbuzz                  1.3.4                         2    conda-forge
hdf5                      1.8.18                        0    conda-forge
icu                       58.1                          1    conda-forge
ipython                   6.1.0                    py36_0    conda-forge
ipython_genutils          0.2.0                    py36_0    conda-forge
jasper                    1.900.1                       4    conda-forge
jedi                      0.10.0                   py36_0    conda-forge
jpeg                      9b                            0    conda-forge
libffi                    3.2.1                         3    conda-forge
libgfortran               3.0.0                         1
libiconv                  1.14                          4    conda-forge
libpng                    1.6.28                        0    conda-forge
libtiff                   4.0.6                         7    conda-forge
libwebp                   0.5.2                         7    conda-forge
libxcb                    1.12                          1    conda-forge
libxml2                   2.9.4                         4    conda-forge
ncurses                   5.9                          10    conda-forge
numpy                     1.12.1          py36_blas_openblas_200  [blas_openblas]  conda-forge
openblas                  0.2.19                        2    conda-forge
opencv                    3.2.0           np112py36_blas_openblas_203  [blas_openblas]  conda-forge
openssl                   1.0.2k                        0    conda-forge
pcre                      8.39                          0    conda-forge
pexpect                   4.2.1                    py36_0    conda-forge
pickleshare               0.7.3                    py36_0    conda-forge
pip                       9.0.1                    py36_0    conda-forge
pixman                    0.34.0                        0    conda-forge
prompt_toolkit            1.0.14                   py36_0    conda-forge
ptyprocess                0.5.1                    py36_0    conda-forge
pygments                  2.2.0                    py36_0    conda-forge
python                    3.6.1                         3    conda-forge
qt                        5.6.2                         3    conda-forge
readline                  6.2                           0    conda-forge
setuptools                33.1.1                   py36_0    conda-forge
simplegeneric             0.8.1                    py36_0    conda-forge
six                       1.10.0                   py36_1    conda-forge
sqlite                    3.13.0                        1    conda-forge
tk                        8.5.19                        1    conda-forge
traitlets                 4.3.2                    py36_0    conda-forge
wcwidth                   0.1.7                    py36_0    conda-forge
wheel                     0.29.0                   py36_0    conda-forge
x264                      20131217                      3    conda-forge
xorg-libxau               1.0.8                         3    conda-forge
xorg-libxdmcp             1.1.2                         3    conda-forge
xz                        5.2.2                         0    conda-forge
zlib                      1.2.11                        0    conda-forge
```

**Comment by patricksnape** [comment]
It's linked against ffmpeg 3.2 I think, so you need to upgrade ffmpeg

**Comment by hajapy** [comment]
Yeah, I think perhaps the requirement should be updated to `ffmpeg 3.2.*`

**Comment by patricksnape** [comment]
That's probably too strict - `3.*` is probably sufficient

**Comment by jakirkham** [comment]
Interestingly the pinning script has [2.8]( https://github.com/conda-forge/conda-forge.github.io/blob/b38af384b7da384566776ccfd10b12bf5746ae0f/scripts/pin_the_slow_way.py#L36 ). Though I agree we shouldn't be using something that old. Maybe we could start by fixing the pinning there? 😉

**Comment by gqmelo** [comment]
Pinning to ```3.*``` is a bad idea. As you can see on https://abi-laboratory.pro/tracker/timeline/ffmpeg/ some 3.* versions add new symbols. You cannot allow to build with 3.3 and run with 3.2.5 for example.

**Comment by patricksnape** [comment]
Ok well then we can fix to 2.8 - I don't have strong feelings about what ffmpeg to use.

**Comment by hajapy** [comment]
Closed by https://github.com/conda-forge/opencv-feedstock/pull/81
